service/src/test/java/uk/gov/hmcts/reform/fpl/model/CaseDataTest.java

```diff
@@ -73,99 +73,94 @@ void shouldDeserialiseCaseDataWhenCaseManagementOrderWithSendToJudgeState() thro
+        Other other2 = otherWithName("Sam");
+
         CaseData caseData = CaseData.builder()
             .others(Others.builder()
                 .firstOther(other1)
-                .additionalOthers(ElementUtils.wrapElements(other2))
+                .additionalOthers(wrapElements(other2))
                 .build())
             .build();
 
-        assertThat(caseData.getAllOthers()).containsExactly(other1, other2);
+        assertThat(caseData.getAllOthers().get(0).getValue()).isEqualTo(other1);
+        assertThat(caseData.getAllOthers().get(1).getValue()).isEqualTo(other2);
     }
 
     @Test
-    public void shouldGetEmptyListOfOthersWhenOthersIsNull() {
+    void shouldGetEmptyListOfOthersWhenOthersIsNull() {
         CaseData caseData = CaseData.builder().build();
+
         assertThat(caseData.getAllOthers()).isEmpty();
     }
 
     @Test
-    public void shouldGetEmptyListOfOthersWhenOthersAreEmpty() {
-        CaseData caseData = CaseData.builder()
-            .others(Others.builder().build())
-            .build();
-        assertThat(caseData.getAllOthers()).isEmpty();
+    void shouldGetEmptyListOfOthersWhenOthersAreEmpty() {
+        CaseData caseData = CaseData.builder().others(Others.builder().build()).build();
+
+        assertThat(caseData.getAllOthers().get(0).getValue()).isNull();
     }
 
     @Test
-    public void shouldGetFirstOtherWhenNoAdditionalOthers() {
-        Other other1 = Other.builder().build();
-        CaseData caseData = CaseData.builder()
-            .others(Others.builder()
-                .firstOther(other1)
-                .build())
-            .build();
-        assertThat(caseData.getAllOthers()).containsExactly(other1);
+    void shouldGetFirstOtherWhenNoAdditionalOthers() {
+        Other other1 = otherWithName("John");
+        CaseData caseData = CaseData.builder().others(Others.builder().firstOther(other1).build()).build();
+
+        assertThat(caseData.getAllOthers().get(0).getValue()).isEqualTo(other1);
     }
 
     @Test
-    public void shouldFindFirstOther() {
-        Other other1 = Other.builder().build();
-        CaseData caseData = CaseData.builder()
-            .others(Others.builder()
-                .firstOther(other1)
-                .build())
-            .build();
+    void shouldFindFirstOther() {
+        Other other1 = otherWithName("John");
+        CaseData caseData = CaseData.builder().others(Others.builder().firstOther(other1).build()).build();
+
         assertThat(caseData.findOther(0)).isEqualTo(Optional.of(other1));
     }
 
     @Test
-    public void shouldNotFindNonExistingOther() {
-        Other other1 = Other.builder().build();
+    void shouldNotFindNonExistingOther() {
+        Other other1 = otherWithName("John");
         CaseData caseData = CaseData.builder()
-            .others(Others.builder()
-                .firstOther(other1)
-                .build())
+            .others(Others.builder().firstOther(other1).build())
```

Comment:
might be worth putting this into private method since we use it 3 times, buildCaseDataWithFirstOther() or something

Reply:
have extracted a caseData method